src/services/DfusionService.ts

```diff
@@ -174,7 +174,7 @@ export class DfusionRepoImpl implements DfusionService {
           this._getToken(sellTokenAddress),
           this._getToken(buyTokenAddress),
           this._batchIdToDate(validFromBatchId),
-          this._batchIdToDate(validUntilBatchId),
+          this._batchIdToDate(validUntilBatchId.plus(1)),
```

Comment:
Just for future reference, when someone is looking at the code and wondering: "why the fuck are we adding 1 here?", could you add a comment explaining it?
Can have a link to the issue where this is discussed, or just a short explanation.

Reply:
```suggestion
          this._batchIdToDate(validUntilBatchId.plus(1)), // adding 1 since validUntillBatchId is inclusive. https://github.com/gnosis/dex-telegram/issues/183
```